Robot_Adapter/Convert/ToRobot/Loads/Load.cs

```diff
@@ -28,15 +28,15 @@
 using BH.oM.Geometry;
 using BH.Engine.Geometry;
 
-namespace BH.Engine.Robot
+namespace BH.Adapter.Robot
 {
     public static partial class Convert
     {
         /***************************************************/
         /****           Public Methods                  ****/
         /***************************************************/
 
-        public static IRobotCaseNature RobotLoadNature(Loadcase lCase, out int subNature)
+        public static IRobotCaseNature ToRobot(Loadcase lCase, out int subNature)
```

Comment:
Maybe tweak name here as well, as it is not a full convert of the Loadcase. Alternatively, change the input to accept just a LoadNature and keep the name.